bundles/org.openhab.binding.venstarthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -89,6 +146,141 @@
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="schedulePart">
+		<item-type>String</item-type>
+		<label>Schedule Part</label>
+		<description>Current Schedule Part</description>
+		<state readOnly="true">
+			<options>
+				<option value="morning">Morning</option>
+				<option value="day">Day</option>
+				<option value="evening">Evening</option>
+				<option value="night">Night</option>
+				<option value="inactive">Inactive</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="schedulePartRaw" advanced="true">
+		<item-type>Number</item-type>
+		<label>Schedule Part (Raw)</label>
+		<description>Current Schedule Part, as an integer number</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="timestampRuntime" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Runtime Record Timestamp</label>
+		<description>Time stamp of Runtime Update (Day 0 = TODAY, Day 6 = 6 days ago)</description>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM"/>
+	</channel-type>
+
+	<channel-type id="heat1Runtime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Heat1 Runtime Minutes</label>
+		<description>Run time in heat1 mode in minutes (Day 0 = TODAY, Day 6 = 6 days ago)</description>
+		<state readOnly="true" pattern="%d mins"/>
+	</channel-type>
+
+	<channel-type id="heat2Runtime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Heat2 Runtime Minutes</label>
+		<description>Run time in heat2 mode in minutes (Day 0 = TODAY, Day 6 = 6 days ago)</description>
+		<state readOnly="true" pattern="%d mins"/>
+	</channel-type>
+
+	<channel-type id="cool1Runtime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Cool1 Runtime Minutes</label>
+		<description>Run time in cool1 mode in minutes (Day 0 = TODAY, Day 6 = 6 days ago)</description>
+		<state readOnly="true" pattern="%d mins"/>
+	</channel-type>
+
+	<channel-type id="cool2Runtime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Cool2 Runtime Minutes</label>
+		<description>Run time in cool2 mode in minutes (Day 0 = TODAY, Day 6 = 6 days ago)</description>
+		<state readOnly="true" pattern="%d mins"/>
+	</channel-type>
+
+	<channel-type id="aux1Runtime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Aux1 Runtime Minutes</label>
+		<description>Run time in aux1 mode in minutes (Day 0 = TODAY, Day 6 = 6 days ago)</description>
+		<state readOnly="true" pattern="%d mins"/>
+	</channel-type>
+
+	<channel-type id="aux2Runtime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Aux2 Runtime Minutes</label>
+		<description>Run time in aux2 mode in minutes (Day 0 = TODAY, Day 6 = 6 days ago)</description>
+		<state readOnly="true" pattern="%d mins"/>
+	</channel-type>
+
+	<channel-type id="freeCoolRuntime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
Why using `Number:Dimensionless` ?
You are providing a DecimalType when updating the channel, so `Number` looks appropriate.

Reply:
OK